plugin_gems.rb

```diff
@@ -18,7 +18,7 @@
 download "string-scrub", "0.0.5"
 download "fluent-plugin-rewrite-tag-filter", "1.5.5"
 download "fluent-plugin-td-monitoring", "0.2.2"
-download "fluent-plugin-google-cloud", "0.7.0"
+download "fluent-plugin-google-cloud", "0.7.2"
```

Comment:
The `fluent-plugin-google-cloud` version inside the package needs to be one version older because this is the current stable version.

The other versions are the `next-to-be-released` version.

Reply:
Ignore this comment. The version in this PR is consistent. I confused it with the other PR.